mpu6050/testing/MeasurePerformance.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+if [[ $1 = "-h" || $1 = "--help" ]]; then
+	echo "-h, --help     display this help and exit"
+	echo "[time] 	     testing time ms (default 1(s))"
+	exit 0
+fi
+
+if [[ ! -n $1 ]]; then
```

Comment:
! -n  is the same like -z
What is the reason for negative condition to if-else block with 2 branches?

Reply:
To select a variable value (default or custom). 
Can be written as:
DEF_TIMEOUT=1000
TIMEOUT=$ {1: - $ DEF_TIMEOUT}

Reply:
Replaced by :
TIMEOUT=${1:-1000}